README.md

```diff
@@ -1,6 +1,15 @@
 # Change Log
 This is the Maleficarum Worker component implementation. 
 
+## [9.1.0] - 2019-01-29
+### Added
+- Add Debugger encapsulator 
+- Add debug trait for Worker Handler. 
+
+When Handler has `\Maleficarum\Worker\Handler\Encapsulator\Debugger` and use `\Maleficarum\Worker\Handler` 
+you can use method `$this->debug(string $message, array $options)` inside the Handler code. In outcome there is an additional debug entry in logs like `[DEBUG] 1. {"message":"This is a test message","time":"2,408634 sek","memory":"0,000584 MB"}`. 
```

Comment:
"seconds" are spelled with a "c" not a "k". The accepted fraction separator in the English notation is a "." not a ",". Please do not mix languages.

Reply:
Changed at 5f3f302cb54de38384d4ba0180bb30fabe25ec7e